synapse/lib/snap.py

```diff
@@ -303,6 +302,18 @@ def getStormRuntime(self, opts=None, user=None):
             return
 
         lops = prop.getLiftOps(valu, cmpr=cmpr)
+
```

Comment:
need one more check in here unfortunately. it's possible or len(lops) to be 0 ( the code underneath just has a 0 loop generator in getLiftRows and everything works out )